Update desktop-changelog.rst (#3592)
mattermost/docs: source/help/apps/desktop-changelog.rst

Desktop Application Changelog
========================================

Release v4.4
----------------------------

**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.4.1>`_

- **v4.4.2, release day TBD**

>  - Fixing an issue on Windows where no notification is received if the app is closed on a channel where the message is posted. `MM-23215 <https://mattermost.atlassian.net/browse/MM-23215>`_

Reviewer: ```suggestion
 - Fixing an issue on Windows where channel was marked as being viewed if the app is closed on a channel where the message is posted. `MM-23215 <https://mattermost.atlassian.net/browse/MM-23215>`_
```

Desktop Application Changelog
========================================

Release v4.4
----------------------------

**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.4.1>`_

- **v4.4.2, release day TBD**
 - Fixing an issue on Windows where no notification is received if the app is closed on a channel where the message is posted. `MM-23215 <https://mattermost.atlassian.net/browse/MM-23215>`_

>  - Fixing an issue on MacOS Catalina where notifications are disabled by default in v4.4 on a fresh install. `MM-24744 <https://mattermost.atlassian.net/browse/MM-24744>`_

Reviewer 2: This is not an issue, it needs UX input and an improvement ticket created
